orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -111,4 +112,8 @@ class CreateWebhookTask implements RetryableTask {
     }
   }
 
+  static def readJsonPath(def json, String jsonPath) {
```

Comment:
personally not a fan of `def` for either a return or parameter type ... can we properly type it?

Reply:
Rather than split this out to a new method I believe you could just change the source to

```
JsonPath path = JsonPath.compile(stage.context.statusUrlJsonPath as String)
statusUrl = new JsonContext().parse(response.body).read(path)
```

Reply:
I `def`'d it because it can return a string/object/array/number.
I'm going to inline this an pretend that it always returns a string